### PR TITLE
feat(iframe): disable bridge inside iframes by default. Can be enabled using a data attribute

### PR DESCRIPTION
**Pull request by SergioMorchon**
For more context, visit this: https://github.com/Telefonica/webapp/pull/736

### CODE REVIEW

**Review by wyunreal — APPROVED** [review]

**Review by atabel — COMMENTED** [review]
We should review why the iframes have access to native app js bridges, because I think they shouldn't have access.

Even if we block this at webview-bridge js lib level, the iframe could still use the bridge by calling the functions exposed by native app

File: src/post-message.ts [hunk 1]
```diff
@@ -258,11 +258,15 @@ const unsubscribe = (listener: MessageListener) => {
     messageListeners = messageListeners.filter((f) => f !== listener);
 };
 
+const isDisabledFromIframe = () =>
+    frameElement?.hasAttribute('data-disable-webview-bridge') ?? false;
```
Comment (pladaria) [inline src/post-message.ts:262]: check that frameElement (or window) exists, otherwise this isn't SSR friendly

Comment (pladaria) [reply src/post-message.ts:262]: Perhaps it would be better to disable by default and explicitly enable it with a data attribute

Comment (atabel) [reply src/post-message.ts:262]: yes, I prefer to disable it by default inside iframes

Comment (SergioMorchon) [reply src/post-message.ts:262]: Sure, I did it this way to ensure zero impact on current consumers.
I'll change it 👍

**Review by pladaria — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**